corehq/apps/userreports/tasks.py

```diff
@@ -365,3 +365,16 @@ def _indicators_by_count(date_created=None):
             ret[config_id] += ind['indicator_config_ids__count']
 
     return ret
+
+
+@periodic_task(
+    run_every=crontab(minute="*/15"),
+    queue=settings.CELERY_PERIODIC_QUEUE,
+)
+def icds_async_indicators_metrics():
+    indicator_count_until_28 = _indicators_by_count(datetime(2017, 6, 28))
```

Comment:
i'm going to use it for a dashboard for a rebuild happening on icds. the 28th is a super safe date to use. meaning that i'm 100% ceratin that everything that matters in the table is before that date. I think I could use a later date, but I don't have anything that can verify what date is the safest..